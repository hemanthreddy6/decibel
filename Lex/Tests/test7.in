audio my_audio <- load "my_file.wav";
play my_audio

modified_my_audio = HIGHPASS(my_audio, 440Hz)
save modified_my_audio -> "modified_audio.wav"