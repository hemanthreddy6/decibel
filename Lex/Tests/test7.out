IDENTIFIER
:
AUDIO
<-
LOAD
STRING_LITERAL
;
PLAY
IDENTIFIER
IDENTIFIER
=
HIGHPASS
(
IDENTIFIER
,
FLOAT_LITERAL
)
SAVE
IDENTIFIER
->
STRING_LITERAL
